lib/t/matches.ex

```diff
@@ -125,7 +137,16 @@ defmodule T.Matches do
         [user_id_1, user_id_2] = Enum.sort(user_ids)
         m = "New match: #{user_id_1} and #{user_id_2}"
         Bot.async_post_message(m)
-        Repo.insert(%Match{user_id_1: user_id_1, user_id_2: user_id_2})
+        {:ok, match} = Repo.insert(%Match{user_id_1: user_id_1, user_id_2: user_id_2})
+
+        Repo.insert(%MatchEvent{
+          timestamp: DateTime.utc_now() |> DateTime.truncate(:second),
+          match_id: match.id,
+          event: "created"
+        })
+
+        expiration_date = expiration_date(match.id)
```

Comment:
alternatively we can just compute it here using DateTime.now + expiration_duration for better performance (but it may hurt consistency a bit)